recipes/igraph-louvain/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{%set name = "louvain-igraph" %}
+{%set version = "0.5.3" %}
+{%set sha256 = "1bc4ea51eed5a6983ef3a1552c3261039502892eae9b4cb075a14ac59ecc8441" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/vtraag/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - igraph
+    - python-igraph
```

Comment:
I don't think this is needed, because python extensions should be compiled with MSVC whenever possible and not with MinGW